design/src/glossary.md

```diff
@@ -39,12 +39,55 @@ Terms defined here are standard elements of the factory builder genre.
 
 ### Belt
```

Comment:
Belts can have sides, which is an important gameplay difference wihin the genre.

Reply:
Do you mean the two-side split in Factorio or the direction the belt is facing?

Reply:
The two-side split.